sentry/src/main/java/io/sentry/SentryUncaughtExceptionHandler.java

```diff
@@ -60,4 +56,22 @@ public void uncaughtException(Thread thread, Throwable thrown) {
         }
     }
 
+    /**
+     * Configures an uncaught exception handler which sends events to
+     * Sentry, then calls the preexisting uncaught exception handler.
+     */
+    public static void setup() {
+        Thread.UncaughtExceptionHandler currentHandler = Thread.getDefaultUncaughtExceptionHandler();
+        if (currentHandler != null) {
+            logger.debug("default UncaughtExceptionHandler class='" + currentHandler.getClass().getName() + "'");
+        }
+
+        // don't double register
+        if (!(currentHandler instanceof SentryUncaughtExceptionHandler)) {
+            // register as default exception handler
+            Thread.setDefaultUncaughtExceptionHandler(
+                new SentryUncaughtExceptionHandler(currentHandler));
+        }
+    }
```

Comment:
Existing code moved/renamed from Android so that any Java user can make use of it.